0_explore_data/cluster_by_accessory_gene.ipynb

```diff
@@ -2024,147 +2024,147 @@
        "  <thead>\n",
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
-       "      <th>PA2004</th>\n",
-       "      <th>PA2174</th>\n",
-       "      <th>PA1173</th>\n",
-       "      <th>PA0101</th>\n",
-       "      <th>PA1242</th>\n",
-       "      <th>PA5400</th>\n",
-       "      <th>PA4374</th>\n",
-       "      <th>PA0174</th>\n",
-       "      <th>PA2155</th>\n",
-       "      <th>PA4871</th>\n",
+       "      <th>PA4655</th>\n",
+       "      <th>PA1271</th>\n",
+       "      <th>PA1705</th>\n",
+       "      <th>PA3140</th>\n",
+       "      <th>PA2312</th>\n",
+       "      <th>PA3601</th>\n",
+       "      <th>PA2697</th>\n",
+       "      <th>PA5275</th>\n",
+       "      <th>PA0791</th>\n",
+       "      <th>PA5410</th>\n",
        "      <th>...</th>\n",
-       "      <th>PA3211</th>\n",
-       "      <th>PA3589</th>\n",
-       "      <th>PA0991</th>\n",
-       "      <th>PA4006</th>\n",
-       "      <th>PA3427</th>\n",
-       "      <th>PA0252</th>\n",
-       "      <th>PA0080</th>\n",
-       "      <th>PA1083</th>\n",
+       "      <th>PA5293</th>\n",
+       "      <th>PA5301</th>\n",
+       "      <th>PA1655</th>\n",
+       "      <th>PA0920</th>\n",
+       "      <th>PA1811</th>\n",
+       "      <th>PA5172</th>\n",
+       "      <th>PA1357</th>\n",
+       "      <th>PA0433</th>\n",
        "      <th>Strain type</th>\n",
        "      <th>median core expression</th>\n",
        "    </tr>\n",
        "  </thead>\n",
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>ERX1272616</th>\n",
-       "      <td>87.578274</td>\n",
-       "      <td>55.796995</td>\n",
-       "      <td>23.440490</td>\n",
-       "      <td>63.993977</td>\n",
-       "      <td>34.513605</td>\n",
-       "      <td>6.327494</td>\n",
-       "      <td>176.163194</td>\n",
-       "      <td>7.477948</td>\n",
-       "      <td>13.949249</td>\n",
-       "      <td>90.598214</td>\n",
-       "      <td>...</td>\n",
-       "      <td>448.820677</td>\n",
-       "      <td>9.203628</td>\n",
-       "      <td>96.638095</td>\n",
-       "      <td>309.328188</td>\n",
-       "      <td>1793.988446</td>\n",
+       "      <td>720.471512</td>\n",
+       "      <td>392.592261</td>\n",
        "      <td>3.307554</td>\n",
-       "      <td>22.577650</td>\n",
-       "      <td>244.039951</td>\n",
+       "      <td>22.865264</td>\n",
```

Comment:
why did these numbers change so much? Is it just an off index, and the actual result didn't change?

Reply:
This ordering of this column is different but the results are the same.